docs/dictionary/message/rawKeyUp.lcdoc

```diff
@@ -29,7 +29,7 @@ The raw keycode of the released key.
 
 The result:
 If the <rawKeyUp> <message> is sent as the result of a keypress, the
-<message> is sent to the <active (focused) control(glossary)>, or to the
+<message> is sent to the <active control(glossary)>, or to the
```

Comment:
As with RawKeyDown, no need for glossary in parenthesis here and can we have this read:

> the \<message\> is sent to the \<focus|focused\> or \<active control\>, or to the \<current card\> if no \<control\> is \<focus|focused\>.

Reply:
To avoid confusion, do we also need this part of the cross platform note as in rawKeyDown:

>*Cross-platform note:*  On <Mac OS|Mac OS systems>, no \<message\> is
> sent when a \<modifier key\> (Shift, Option, Control, or Command) is
> pressed, unless another key is pressed along with the \<modifier key\>.